apps/src/p5lab/AnimationTab/PiskelEditor.jsx

```diff
@@ -29,7 +29,7 @@ class PiskelEditor extends React.Component {
     // Provided by Redux
     animationList: shapes.AnimationList.isRequired,
     currentAnimation: shapes.AnimationKey,
-    channelId: PropTypes.string.isRequired,
+    channelId: PropTypes.string,
```

Comment:
I think channel ID is unused in this file right? should we just go ahead and remove it while we're here?

Reply:
It looks like it, but I was unsure about how the PiskelEditor works under the hood so I decided not to remove it in order not to break anything by accident